.gitignore

```diff
@@ -49,7 +49,7 @@ orc8r/cloud/test_certs/*.srl
 orc8r/cloud/test_certs/*.pfx
 
 # Ignore generated Swagger files
-orc8r/cloud/configs/swagger_specs
+orc8r/swagger/specs
```

Comment:
Let's actually do orc8r/cloud/swagger/specs, just to clarify that this is for cloud not gateway